composer_multi_project_monitoring/main.tf

```diff
@@ -0,0 +1,1018 @@
+# [START composer_multi_project_monitoring]
+#   Monitoring for multiple Cloud Composer environments
+#   
+#   Usage:
+#       1. Create a new project that you will use for monitoring of Cloud Composer environments in other projects
+#       2. Store the name of this project in "monitoring_project" variable below, in the locals block
+#       3. Save the list of projects with Cloud Composer environments to be monitored in "monitored_projects", set in the locals block
+#       4. Run "terraform apply"
+#
+#   The script creates the following resources in the monitoring project:
+#           1. Adds monitored projects to Cloud Monitoring
+#           2. Creates Alert Policies
+#           3. Creates Monitoring Dashboard
+#
+
+
+#######################################################
+#  
+# Overall settings
+#
+########################################################
+
+
+locals {
+    monitoring_project = "YOUR_MONITORING_PROJECT"
+    monitored_projects = toset(["YOUR_PROJECT_TO_MONITOR_1", "YOUR_PROJECT_TO_MONITOR_2", "YOUR_PROJECT_TO_MONITOR_3"])
+}
+
+#######################################################
+#  
+# Provider
+#
+########################################################
+
+provider "google-beta" {
+  project = local.monitoring_project
+}
+provider "google" {
+  project = local.monitoring_project
+}
+
```

Comment:
nit: provider initialization should not be in samples

Reply:
Disclaimer I'm someone who does not write TF scripts often and am not the original author of this one - if it's not in the sample, Where should it be? Is it something a user will configure elsewhere?

Reply:
@leahecole I believe the guide refers to adding this when needed to test a sample. Users will refer to https://cloud.google.com/docs/terraform/basic-commands which is usually linked alongside a sample.

Reply:
SGTM, removed.